acp/source_hub_client.go

```diff
@@ -276,7 +276,7 @@ func (a *sourceHubBridge) RegisterDocObject(
 	resourceName string,
 	docID string,
 ) error {
-	registerDocResult, err := a.client.RegisterObject(
+	err := a.client.RegisterObject(
```

Comment:
thought: I think at this point can just return `a.client.RegisterObject(...)` directly